app/src/main/res/color/overlay_button_border.xml

```diff
@@ -5,6 +5,7 @@
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:state_pressed="true" android:color="@color/button_pressed" />
-    <item android:state_focused="true" android:color="@color/button_focused" />
+    <item android:state_focused="true" android:color="@color/tv_blue50" />
+    <item android:state_checked="true" android:color="@color/button_checked"/>
```

Comment:
Do you need to include `checked` here? I think it should just fall through to the default color (below).

Reply:
Yes- the default border is transparent, but the checked border needs to be black.